h_program-lang/skip_code.ml

```diff
@@ -67,8 +67,7 @@ let load file =
 (* Main entry point *)
 (*****************************************************************************)
 
-(* less: say when skipped stuff? *)
-let filter_files skip_list root xs =
+let filter_files skip_list ~root relative_paths : string list * string list =
```

Comment:
Might be useful to have a dedicated (record) type to use instead of `string list * string list`.

Reply:
meh. It's a yes/no split, similar to `List.partition`.

Reply:
This return type is used in many places, and the type doesn't tell you what is what. IMO it is error prone and a record type is very cheap. Out of curiosity, in Core `List.partition` is called `List.partition_tf` (last time I checked at least).

Reply:
It's a small detail anyways, just wanted to clarify my suggestion. For me it's always a red flag when a function takes or returns more than one built-in type `int`, `string`, `int list`, and such. I've seen quite a few bugs using such functions that would have been caught using a new type or a label.